wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -108,50 +125,78 @@ class MainActivity : ComponentActivity() {
 
     private val provisioningViewModel: ProvisioningViewModel by viewModels()
     private val credentialInformationViewModel: CredentialInformationViewModel by viewModels()
+    private lateinit var sharedPreferences: SharedPreferences
 
     private val permissionTracker = PermissionTracker(this, mapOf(
```

Comment:
Just create a single permission tracker for all permissions. Map may require a builder function.

Reply:
the reason I have the `inPersonPresentationPermissionTracker` separate from the camera permissions in `permissionTracker` is so that they can be requested at startup - see David's comment [here](https://github.com/openwallet-foundation-labs/identity-credential/pull/459#discussion_r1479986172)